src/components/Chart/Chart.js

```diff
@@ -1,13 +1,12 @@
 import React, { Component, Fragment } from 'react'; 
 import { min, max, quantile } from 'd3-array';
-import { scaleLinear, scaleBand, scaleLog, scalePow } from 'd3-scale';
+import { scaleLinear, scaleBand, scalePow } from 'd3-scale';
 import { select } from 'd3-selection';
 import { easeCubicOut } from 'd3-ease'
-import { timeFormat } from 'd3-time-format';
 import { Tooltip } from 'antd';
 import Axis from '../Graph/Axis';
 import { getDateIdx, addCommas, capitalize } from '../../utils/utils';
-import { margin, chartBkgd, green, gray, blue, scenarioColors } from '../../utils/constants'
+import { margin, chartBkgd, gray, blue, scenarioColors } from '../../utils/constants'
```

Comment:
Another way to make this import easier is to shove all colors into a single const and file.

Example:

```
const colors = {
  red: 'red',
  blue: 'blue',
  purple_nurple: '#123456'
}
```

Then when you use it its a single import.

```
import {colors} from ../../utils/colors

const colorVariable = colors.purple_nurple
```